README.md

```diff
@@ -1,62 +1,63 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Porfolio
```

Comment:
- [ ] Your linters are not passing all the tests 😨 Kindly make your linters pass all the tests and submit your pull request again from your dashboard 👍

Reply:
@HAMMAS-SALEEM 
The problem above was [resolved](https://github.com/Lucash2022/LucasErkanaPortfolio_Website/commit/63033c8233bef57703eb51cf558144ddb61367d7).